administrator-manual/en/webtop5.rst

```diff
@@ -331,6 +362,58 @@ A precompiled wizard will open:
 
 Proceed with the Next button until the Wizard is complete.
 
+Use the personal Cloud to send and receive documents
+====================================================
+
+Using Cloud module allows you to send and receive documents throug web links.
+
+.. note::
+
+   The server must be reachable in http on port 80   
```

Comment:
Add a empty line here before the next title.

Reply:
done